### ISSUE TITLE
Loci that attest a lexeme twice shouldn't be listed twice in lexeme show

### ISSUE THREAD

**Issue by kohath** [Rexko/rexko#101]
Currently if a locus attests a lexeme more than once, it'll be listed multiple times, but it's only helpful to list it once.

(Note that when #48 is fixed, it should still be _numbered_ for each appearance, but still listed only once.)

**Comment by Muskworker** [comment]
This is a duplicate of #69 and was resolved WAY back in ff405071ca396ba2d6ca50457b249d4498c94912.
